qualtran/bloqs/qft/approximate_qft_test.py

```diff
@@ -70,7 +69,6 @@ def test_approximate_qft_exact(n: int, without_reverse: bool):
     qft_bloq = TestApproximateQFT(n, n, not without_reverse)
     qft_cirq = cirq.QuantumFourierTransformGate(n, without_reverse=without_reverse)
     np.testing.assert_allclose(cirq.unitary(qft_bloq), cirq.unitary(qft_cirq))
-    np.testing.assert_allclose(cirq.unitary(qft_bloq**-1), cirq.unitary(qft_cirq**-1))
```

Comment:
Why was this removed? Does this assertion not pass anymore?

Reply:
added back after adding inverse implementation for AddIntoPhaseGrad